tests/lib/edge_impulse/src/main.cpp

```diff
@@ -430,7 +431,7 @@ static void test_data_isr(void)
 	zassert_ok(err, "Cannot take semaphore");
 }
 
-static void setup_fn(void)
+static void setup_fn(void*)
```

Comment:
btw. should we name the argument here? You may also use `ARG_UNUSED` to clearly mark that it's not going to be used in the function.